src/screens/regionPicker/RegionPicker.tsx

```diff
@@ -0,0 +1,126 @@
+  {code: 'NT', flagIcon: NtFlag, name: 'Northwest Territories'},
+  {code: 'NS', flagIcon: NsFlag, name: 'Nova Scotia'},
+  {code: 'NU', flagIcon: NuFlag, name: 'Nunavut'},
+  {code: 'ON', flagIcon: OnFlag, name: 'Ontario'},
+  {code: 'PE', flagIcon: PeFlag, name: 'Prince Edward Island'},
+  {code: 'QC', flagIcon: QcFlag, name: 'Quebec'},
+  {code: 'SK', flagIcon: SkFlag, name: 'Saskatchewan'},
+  {code: 'YT', flagIcon: YtFlag, name: 'Yukon'},
+];
+
+const RegionItem_ = ({code, onPress, name, flagIcon, selected}: RegionItemProps) => (
+  <>
+    <TouchableOpacity onPress={() => onPress(code)}>
+      <Box paddingVertical="s" flexDirection="row" alignContent="center" justifyContent="space-between">
+        <Box flexDirection="row" alignItems="center" paddingVertical="s">
+          <Image source={flagIcon} style={styles.flag} />
+          <Text variant="bodyText" color="overlayBodyText" marginHorizontal="m">
+            {name}
+          </Text>
+        </Box>
+        <Box alignSelf="center">{selected && <Icon size={32} name="icon-check" />}</Box>
+      </Box>
+    </TouchableOpacity>
+    <Box height={1} marginHorizontal="-m" backgroundColor="overlayBackground" />
+  </>
+);
+const RegionItem = React.memo(RegionItem_);
+
+export const RegionPickerScreen = () => {
+  const [i18n] = useI18n();
+  const {setRegion: persistRegion} = useStorage();
+  const [selectedRegion, setSelectedRegion] = useState<Region>('None');
+  const {navigate} = useNavigation();
+
+  return (
+    <Box flex={1} backgroundColor="overlayBackground">
+      <SafeAreaView style={styles.flex}>
+        <ScrollView style={styles.flex}>
+          <Box flex={1} paddingHorizontal="m">
+            <Text variant="bodySubTitle" color="overlayBodyText" textAlign="center">
+              Select your province or territory
+            </Text>
+            <Text marginVertical="m" variant="bodyText" color="overlayBodyText" textAlign="center">
+              This helps give you more relevant guidance.{'\n'}This information is only stored on your device and will
+              never be shared with anyone else.
+            </Text>
+            <Box paddingHorizontal="m" borderRadius={10} backgroundColor="infoBlockNeutralBackground">
+              {items.map(item => {
+                return (
+                  <RegionItem
+                    key={item.code}
+                    selected={selectedRegion === item.code}
+                    onPress={setSelectedRegion}
+                    {...item}
+                  />
+                );
+              })}
+            </Box>
+          </Box>
+        </ScrollView>
+        <Box
+          backgroundColor="overlayBackground"
+          padding="m"
+          borderTopColor="infoBlockNeutralBackground"
+          borderTopWidth={1}
+        >
+          <Button
+            text={selectedRegion === 'None' ? 'Skip' : 'Get started'}
+            variant={selectedRegion === 'None' ? 'bigHollow' : 'bigFlat'}
```

Comment:
Unsure which variant of button is used for the `Skip` button

Reply:
This looks right to me